client/src/components/registration/RegistrationForm.js

```diff
@@ -85,22 +86,32 @@ const RegistrationForm = () => {
   }
 
   return (
+    <div className="form">
+    <div className="trail-bg-img">
     <div className="grid-container" onSubmit={onSubmit}>
       <h1>Register</h1>
       <form>
         <div>
           <label>
             Email
-            <input type="text" name="email" value={userPayload.email} onChange={onInputChange} />
+            <input 
+            className="textboxclass" 
+            type="text" 
+            name="email" 
+            placeholder="Email"
+            value={userPayload.email} 
+            onChange={onInputChange} />
```

Comment:
properties should be indented/ nested under the `input`

Reply:
May be the work prior to "prettier", hoping that it saves what we need this time. ALL of my extensions were not working up until this afternoon.